README.md

```diff
@@ -20,10 +20,10 @@ by a [supported web framework](#supported-web-frameworks):
 
 ```bash
 # Install it generally:
-$ gem install rack
+$ gem install rack --version 3.0.0.beta1
```

Comment:
I think we can use `gem install rack --pre` in these cases.  That way we shouldn't need an update for `rc1`.

Reply:
Thanks, that makes total sense to me.